graph/graph-options.go

```diff
@@ -26,6 +26,7 @@ type GraphOptions struct {
 	OracleCompareSync     bool    // Compares to oracle results on every iteration, when using a synchronous strategy.
 	AllowAsyncVertexProps bool    // If true, a query views vertex properties asynchronously; faster, but may be inconsistent for certain algorithms. For PageRank, this strategy works well. Requires a concurrent finish.
 	NoConvergeForQuery    bool    // If true, will not finish convergence of the algorithm for the query; useful for an algorithm like PageRank.
+	AlgTimeIncludeQuery   bool    // If true, will include time spent on process query in algorithm time
```

Comment:
I think it makes sense to include this as an available flag.

Reply:
Done!